src/ninja.cc

```diff
@@ -94,6 +94,9 @@ struct NinjaMain : public BuildLogUser {
   /// Build configuration set from flags (e.g. parallelism).
   const BuildConfig& config_;
 
+  /// Client of jobserver to manage job slots assigned to ninja.
+  Jobserver jobserver_;
```

Comment:
This declaration calls the constructor of Jobserver, but jobserver-posix.cc is never compiled for windows. Why don't we get a linker error on Windows?

Reply:
constructor/destructor is blocked off by preprocessor macros for Windows in the header, but like we discussed I can assign NULL pointer here instead